docs/timelines.md

```diff
@@ -0,0 +1,141 @@
+---
+id: timelines
+title: Timelines introduction
+sidebar_label: Timelines introduction
+---
+
+Timelines allow you to create a precisely timed light show which you can either
+run from the console's internal timer or synchronise to an external timecode
+source. This can be useful for an opening sequence for a corporate
+event, a complex song in a show where the song is pre-recorded or run to 
+a click track or timecode, or a multimedia event run to a playback track. You
+can also use it to create [complicated one-shot effects](./timelines/timeline-options.md#fader-tab) for busking.
+
+> You can also create timed shows by creating timecoded cues in 
+[cue-lists](./cue-lists/cue-list-timing.md#running-a-cue-list-to-timecode), but Timeline 
+recording allows much easier creation and editing of a timed show.
+
+Timelines can be built either from cues or chases you have already recorded, or by creating
+new cues as you go along. You can record them in real time by operating the console
+or by adding cues manually at specific times. You can think of a timeline as an automatic replay
+system to push the faders and buttons on the console. The actions recorded in a timeline
+are called **triggers**.
+
+To show the Timeline View window for a playback containing a timeline, touch the playback 
```

Comment:
I think "Playback containing a timeline" might introduce a misleading concept.
```suggestion
To show the Timeline View window for a timeline, touch the playback 
```
"Playback information area" also seems a bit odd but so long as that is language used else where in the manual it's fine.